_posts/2020-08-21-release.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 21 August 2020 - FieldTrip version 20200821 has been released
+categories: [news, release]
+tweet: FieldTrip version 20200821 was just released. See http://www.fieldtriptoolbox.org/#21-August-2020
+---
+
+### 21 August, 2020
+
+FieldTrip version [20200821](http://github.com/fieldtrip/fieldtrip/releases/tag/20200821) has been released.
+See [GitHub](https://github.com/fieldtrip/fieldtrip/compare/20200826...20200821) for the detailled list of updates.
```

Comment:
Yup, this is also incorrect. It is `next to current` rather than `prev to current`